recipes/scip/all/conanfile.py

```diff
@@ -76,11 +80,10 @@ def config_options(self):
 
     def requirements(self):
         if self.options.with_gmp:
-            self.requires("gmp/6.2.1")
+            self.requires("gmp/6.3.0")
         if self.options.with_sym == "bliss":
             self.requires("bliss/0.77")
-        self.requires("soplex/6.0.3")
-        self.requires("zlib/1.2.13")
+        self.requires(f"soplex/{self.soplex_version_belonging_to_me[self.version]}")
```

Comment:
If this is 1 to 1, couldn't we use `self.version` as the version? Or are you opening the door to possibly using newer patches?

Reply:
It is 1-to-1, but Soplex 6.0.4 belongs to SCIP 8.0.4, so versions are not exactly the same.

Reply:
That makes sense, thanks for the extra insight :)